### PR TITLE
Update message on spinner

### PR DESCRIPTION
**Pull request by zdlm**
@Tradeshift/ui

Fixes issue #796

### CODE REVIEW

File: src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/covers/ts.ui.CoverSpirit.js [hunk 1]
```diff
@@ -75,11 +75,15 @@ ts.ui.CoverSpirit = (function using(chained, Client) {
...
-					ts.ui.SpinnerSpirit.summon(message, {
-						color: this.blocking() ? 'rgb(255,255,255)' : 'rgb(85,85,85)'
-					})
-				);
+				if (this._spinner) {
+					this._spinner.text(message);
+				} else {
+					this.dom.append(
+						(this._spinner = ts.ui.SpinnerSpirit.summon(message, {
+							color: this.blocking() ? 'rgb(255,255,255)' : 'rgb(85,85,85)'
```
Comment (esmailpour-hosein) [inline src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/covers/ts.ui.CoverSpirit.js]: Can we put those colors in variables?

File: src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/ts.ui.Spirit.js [hunk 2]
```diff
@@ -51,6 +51,17 @@ ts.ui.Spirit = (function using(Type, GuiArray, confirmed, chained) {
 				this.busy(false);
 			}),
 
+			/**
+			 * Change message.
+			 * @param @optional {string} message
+			 * @returns {this}
+			 */
+			spin: chained(function(message) {
```
Comment (esmailpour-hosein) [inline src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/ts.ui.Spirit.js]: I am not a fan of this method name 🤔 Can we use more expressive name for it? What do you think ?

Comment (zdlm) [reply src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/ts.ui.Spirit.js]: I used spinText before. But also not good. I don't want to use message(), text(). Seems to general. It should be only for the spinner. What do you think, Any good ideas?

Comment (esmailpour-hosein) [reply src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/ts.ui.Spirit.js]: I see 🤔 How about `spinnerMessage`?

**Pull request merged**